modules/manage/pages/security/fips-compliance.adoc

```diff
@@ -0,0 +1,62 @@
+
+== Prerequisites
+
+Before configuring brokers to run in FIPS compliance mode (FIPS mode), check to make sure the `redpanda-rpk-fips` and `redpanda-fips` packages are xref:deploy:deployment-option/self-hosted/manual/production/dev-deployment.adoc#install-redpanda-for-fips-compliance[installed]. These packages are required by both the `redpanda` and `redpanda-tuners` install packages.
+
+== Configure FIPS
+
+When you configure a broker to run in FIPS mode:
+
+* FIPS compliance is enforced _immediately_ upon the startup of Redpanda.
+
+* Redpanda and its dependencies only use FIPS-validated cryptographic modules for all cryptographic algorithms used in a security context.
+
+Redpanda logs an error and exits immediately if:
+
+* The underlying operating system and crypto module are not running in FIPS mode.
+
+* The underlying cryptography module enters into an error state.
+
+* It cannot detect a FIPS-validated library.
+
+To place a broker in FIPS compliance mode, enable `fips_mode` in the Redpanda broker configuration file (typically located in `/etc/redpanda/redpanda.yaml`). All fields are within the `redpanda` object:
+
+```bash
+redpanda:
+  # ....
+  fips_mode: enabled
+  opens
+```
+Available `fips_mode` values are:
+
+* `disabled`: Redpanda is not running in FIPS compliance mode. 
+
+* `enabled`: When Redpanda starts up, it looks for a value of `1` in the file `/proc/sys/crypto/fips_enabled`. If the file doesn't exist or doesn't contain `1`, Redpanda logs an error and exits immediately.
+
+* `permissive`: This setting is a safety value option only, and _should not be used in a production environment_. If specified, Redpanda logs a WARNING, but continues operations even if the underlying operating system is not configured for FIPS. If set, your Redpanda instance is _not_ running in FIPS compliance mode.
+
+=== FIPS SSL configuration
+
+You can specify the following SSL configurations for brokers you wish to run in FIPS compliance mode: 
+
+* `openssl_config_file`: Specifies the path to the OpenSSL configuration file that was created as part of the `redpanda-fips` package installation.  This file is used when OpenSSL is initialized to find the `fipsmodule.cnf` file that was created by the `openssl fipsinstall` command. Suggested value when in FIPS mode: `/opt/redpanda/openssl/openssl.cnf`.
+
+* `openssl_module_directory`: Specifies the path to the directory that contains the `fips.so` cryptographic provider. Suggested value when in FIPS mode: `/opt/redpanda/lib/ossl-modules/`.
+
+ The following configuration starts Redpanda in FIPS mode: 
+ 
+```bash
+redpanda:
+  # ....
+  fips_mode: enabled
+  openssl_config_file: /opt/redpanda/openssl/openssl.cnf
+  openssl_module_directory: /opt/redpanda/lib/ossl-modules/
```

Comment:
```suggestion
 
```bash
redpanda:
  # ....
  fips_mode: enabled
  openssl_config_file: /opt/redpanda/openssl/openssl.cnf
  openssl_module_directory: /opt/redpanda/lib/ossl-modules/
```
  
```

Reply:
^^ Not sure why the closing backticks don't take here